x-pack/plugin/downsample/src/main/java/org/elasticsearch/xpack/downsample/TransportDownsampleAction.java

```diff
@@ -458,18 +454,6 @@ protected void masterOperation(
         }));
     }
 
-    private boolean isForceMergeEnabled(Map<String, Object> sourceIndexMappings) {
-        if (sourceIndexMappings.containsKey("_meta")) {
-            if (sourceIndexMappings.get("_meta") instanceof Map<?, ?> metadataMap) {
-                var enabledForceMergeValue = metadataMap.get("downsample.forcemerge.enabled");
```

Comment:
Just double-checking: do we have any documentation on this "setting"? It's in a weird place so I doubt we care much about it, but just wanted to double-check that we're ok with not supporting this configuration any more.

Reply:
This setting is not documented on purpose. We added it to allow users that cannot upgrade yet from `8.19`, to test the impact of skipping the force-merge request. It's just a workaround since the proper way, which we implement with this PR, cannot be backported.